common/source/docs/common-mro-uavcan-adapter-node.rst

```diff
@@ -0,0 +1,65 @@
+.. _common-mro-uavcan-adapter-node:
+
+=============================
+mRobotics UAVCAN Adapter Node
+=============================
+
+
+The mRo CAN Node is a collaborative project with Ardupilot. It began with a desire to make CAN peripherals more standard in the drone community. Compared to other boards that have open-source hardware but closed source code, mRo CAN node is completely open. The CAN Node F303 is the version 2 of mRobotics' CAN Node, featuring more flash space.
+
+With this board, you can easily utilize the Ardupilot driver library to convert ArduPilot supported GPS, I2C, and GPIO based (ie Safety Switch) peripherals to UAVCAN bus peripherals.
+
+.. image:: ../../../images/mRo-can-node.jpg
+
+
+mRo CAN Node uses open-source code, which utilizes the existing and proven Ardupilot UAVCAN driver library (see :ref:`UAVCAN Adapter Nodes <common-uavcan-adapter-node>` for firmware versions and capabilities). It features a  bootloader for "over CAN" firmware updates and an RTOS running Chibios.
+
+The adapter's CAN Node ID can be set manually or automatically (Not yet supported in stable Ardupilot). The hardware design is open source, so that anyone can contribute to the CAN peripheral world with a proven CAN / MCU interface.
+
+As delivered, it is setup to be plug-and-play as a UAVCAN GPS/Magnetometer/LED/Airspeed by simply plugging in the 6pin GH connector from any mRo or other GPS, and connecting I2C Compasses, RGB LEDs, and/or Airspeed sensors supported by ArduPilot, as desired. 
+
+You can change the firmware to ? in https://firmware.ardupilot.org/AP_Periph if you wish. See the :ref:`SLCAN section of UAVCAN advanced setup<common-uavcan-setup-advanced>` for downloading instructions.
```

Comment:
I am not sure what should go here? Maybe, "There are several peripheral focused firmware to choose from at:

Reply:
fixed